### ISSUE TITLE
transition bounce don't work

### ISSUE THREAD

**Issue by psdon** [Maronato/vue-toastification#6]
It don't work even I specify it via this:
```
Vue.use(Toast, {
  transition: "bounce",
  maxToasts: 20,
  newestOnTop: true
});
```

And looks like the demo app shows the bug.

### How to reproduce (found this unexpectedly)
1. Visit https://maronato.github.io/vue-toastification/
2. Scroll down and find `transition` drop down menu. The default one should indicated `Default bounce`
3. Click it and change to `Custom Fade`
4. Change it back to `Default Bounce`
5. Then click `Show toast`
6. The toast notification will not bounce

**Comment by Maronato** [comment]
Sorry, I added a namespace to the transitions and forgot to update the demo. The name is now [Vue-Toastification__bounce](https://github.com/Maronato/vue-toastification#plugin-registration-vueuse).

I'll update it in a moment.

That said, the value is defined by default, so you shouldn't need to set it manually.

**Comment by Maronato** [comment]
Fixed in 0934d92632df35e29dfffa8cbf256d8a80474a93
